### PR TITLE
Update binary_sensor.command_line.markdown

### PR DESCRIPTION
**Pull request by emontnemery**
**Description:**
Clarify why it's preferred to enclose command string in single quotes and align the examples.
Related: #8069 

## Checklist:

- [x] Branch: Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by emontnemery** [comment]
@bphermansson, @Konstigt: Do you think this is clearer?

File: source/_components/binary_sensor.command_line.markdown [hunk 1]
```diff
@@ -23,12 +23,12 @@ To use your Command binary sensor in your installation, add the following to you
...
 binary_sensor:
   - platform: command_line
-    command: cat /proc/sys/net/ipv4/ip_forward
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
 ```
 
 {% configuration %}
 command:
-  description: The action to take to get the value.
+  description: The action to take to get the value. It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.command_line.markdown]:
```suggestion
  description: The action to take to get the value. It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
```

Will solve the build error or remove the `:` after "double it"

File: source/_components/binary_sensor.command_line.markdown [hunk 2]
```diff
@@ -23,12 +23,12 @@ To use your Command binary sensor in your installation, add the following to you
 # Example configuration.yaml entry
 binary_sensor:
   - platform: command_line
-    command: cat /proc/sys/net/ipv4/ip_forward
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
 ```
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.command_line.markdown]: Why do not you put the warning / note here? With this: `<p class='note'> </p>?`

Like:
```suggestion
<p class='note'>
It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
</p>
```
I wonder if you should put the (above) text in the description of the configuration, I think this is not really the intention.

File: source/_components/binary_sensor.command_line.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_components/binary_sensor.command_line.markdown]: @klaasnicolaas Your suggested change has no change, can you try again?

File: source/_components/binary_sensor.command_line.markdown (hunk 2 above)
Comment (emontnemery) [reply source/_components/binary_sensor.command_line.markdown]: You mean move the warning / note from the configuration variable description to the basic example?

File: source/_components/binary_sensor.command_line.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/binary_sensor.command_line.markdown]: What do you mean with try again?

File: source/_components/binary_sensor.command_line.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_components/binary_sensor.command_line.markdown]: Change it to my suggestion, the warning is not something to put in the description of a variable.

File: source/_components/binary_sensor.command_line.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_components/binary_sensor.command_line.markdown]: I mean, your suggested change is same as what is already there. Maybe github is being buggy.. Can you explain how I should change to solve the build error again?

File: source/_components/binary_sensor.command_line.markdown (hunk 2 above)
Comment (emontnemery) [reply source/_components/binary_sensor.command_line.markdown]: OK

File: source/_components/binary_sensor.command_line.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_components/binary_sensor.command_line.markdown]: nvm, moved the warning as you suggest below

**Review by klaasnicolaas — APPROVED** [review]
Think it is now better, [see also here](https://deploy-preview-8119--home-assistant-docs.netlify.com/components/binary_sensor.command_line/)

**Comment by emontnemery** [comment]
Yeah, agree.
Do you think the language is OK btw, it does sound a bit off:
> It’s highly recommended to enclose the command in single quotes ' as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: ''.

The point is that yaml has many ways of specifying strings and:
- Unquoted strings are tricky as they may be interpreted as other type than string
- Double quoted strings allow escaping, which is probably not desired

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery! 👍

**Pull request merged**